### DISCUSSION TITLE
Html semantic elements support for old browser with polyfills

### DISCUSSION THREAD

**Discussion by MejanH** [vercel/next.js#15836] [Help] [answered]
I found in next Js blog [article](https://nextjs.org/blog/next-9-3#32-kb-smaller-runtime-15-kb-gzip) that next js provide old browser support with polyfills. Does that mean it supports HTML semantic elements, other js syntax like intersection observer API etc.?

**Comment by jamesmosier** [comment, marked as answer]
Next.js itself works in IE11 (and newer) browsers https://nextjs.org/docs/basic-features/supported-browsers-features

Next provides a few [polyfills](https://nextjs.org/docs/basic-features/supported-browsers-features#polyfills) for user code but any additional polyfills/support for older browsers within your own code would need to be polyfilled independently by you. 

Next provides [these JS polyfills](https://github.com/vercel/next.js/blob/canary/packages/next-polyfill-nomodule/src/index.js) for older browsers by default.

Next does not polyfill **all** newer HTML/CSS/JS features for older browsers, so you would need to bring your own polyfills for certain features.
